src/commands/remoteconfig-rollback.ts

```diff
@@ -0,0 +1,56 @@
+import { Command } from "../command";
+import { FirebaseError } from "../error";
+import { prompt } from "../prompt";
+import { requireAuth } from "../requireAuth";
+import { rollbackTemplate } from "../remoteconfig/rollback";
+import { requirePermissions } from "../requirePermissions";
+import { getVersions } from "../remoteconfig/versionslist";
+
+import getProjectId = require("../getProjectId");
+
+module.exports = new Command("remoteconfig:rollback")
+  .description(
+    "Roll back a project's published Remote Config template to the one specified by the provided version number"
```

Comment:
Again, don't capitalize the description

```suggestion
    "roll back a project's published Remote Config template to the one specified by the provided version number"
```